flang/include/flang/Parser/parse-tree.h

```diff
@@ -5243,38 +5243,65 @@ struct AccEndBlockDirective {
 };
 
 // ACC END ATOMIC
-EMPTY_CLASS(AccEndAtomic);
+struct AccEndAtomic {
+  WRAPPER_CLASS_BOILERPLATE(AccEndAtomic, Verbatim);
+  CharBlock source;
```

Comment:
the `Verbatim` parse tree node has a `source` data member.  That is its point.  This new class doesn't add any value.